lib/web_ui/pubspec.yaml

```diff
@@ -18,7 +18,6 @@ dependencies:
     path: ../../third_party/web_test_fonts
 
 dev_dependencies:
-  analyzer: 5.2.0
```

Comment:
Not sure why this is being removed.

Reply:
This dependency is unused.

Reply:
What about the IDE? Which analyzer does it use when I open the `lib/web_ui` folder? For some reason, I always assumed the IDE used the `analyzer` dependency version, but I'm not 100% sure.

Reply:
It uses the analyzer of the dart SDK configured in the IDE. This dependency is used for ‘package:analyzer/…’ imports of which there don’t seem to be any within web_ui.

Reply:
Thanks for clarifying!